.travis.yml

```diff
@@ -1,14 +1,17 @@
-sudo: false
-language: java
-dist: trusty
-jdk: oraclejdk8
-cache:
-  directories:
-  - "$HOME/.m2"
-install: true
-script:
-- mvn clean verify
-- sed -i'' -e "s/\$VERSION/${TRAVIS_TAG:=0.0.0}/" descriptor.json
-- sed -i'' -e "s/\$DATE/$(date +%Y-%m-%d)/" descriptor.json
-notifications:
-  webhooks: https://fathomless-fjord-24024.herokuapp.com/notify
+dist: bionic
+
+language: java 
+jdk:
+  - oraclejdk11

```

Comment:
remove `oraclyjdk11` add `openjdk8`

Reply:
OpenJDK 8 not possible because newest Tycho/Maven/some Plugin there version does not support 8 anymore

Reply:
Then keep it at jdk11-openjdk.